spec/features/navigation_spec.rb

```diff
@@ -33,5 +47,38 @@
       end
 
     end
+
+    it "I can see a profile link if logged in on all pages" do
+      visit '/merchants'
+
+      within 'nav' do
+        expect(page).to have_link("Profile")
+      end
+    end
+
+    it "I can see a logout link if logged in on all pages" do
+      visit '/merchants'
+
+      within 'nav' do
+        expect(page).to have_link("Logout")
+      end
+    end
+
+    it "I cannot see a login or register link if logged in on all pages" do
+      visit '/merchants'
+
+      within 'nav' do
+        expect(page).to_not have_link("Login")
+        expect(page).to_not have_link("Register")
+      end
+    end
+
+    it "I can see 'logged in as' message if logged in on all pages" do
```

Comment:
This test meets us3 requirements for flash message "logged in as <dynamic name>"